man/systemd.network.xml

```diff
@@ -3042,6 +3042,25 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[QuickFairScheduler] Section Options</title>
+    <para>The <literal>[QuickFairScheduler]</literal> section manages the queueing discipline
+    (qdisc) of Quick Fair Scheduler (QFQ).</para>
```

Comment:
QFQ is "Quick Fair Queueing (Plus) Scheduler".

Reply:
Plus is something different. https://access.redhat.com/solutions/1459623

Reply:
OK. Regarding https://en.wikipedia.org/wiki/Network_scheduler#Algorithms, let's rename this to "Quick Fair Queueing", as "Scheduler" is redundant and applicable for all other algorithms.